### PR TITLE
Added editorconfig as a Dependency

### PR DESCRIPTION
**Pull request by JoshQuaintance**
I noticed that there is a `.editorconfig` on the repo, but it won't work without the depedency. Just adding it.

### CODE REVIEW

**Comment by bradtaniguchi** [comment]
I realized this "works" on my end on my machines when I use VSCode with [this extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig). I'm not 100% sure what the npm package does, but I don't think we need it if you use any of the following editors listed as this site:

https://editorconfig.org/#download

**Comment by naomi-lgbt** [comment]
And if you have the necessary plugin. I don't, apparently - which explains why I was battling that line break rule for so long. 
Adding the editorconfig dependency seems to do the same thing as adding the extension, so including it in the package.json would just help catch contributors who don't have the plugin.

**Comment by bradtaniguchi** [comment]
So with the npm extension the editorconfig magically runs with whatever editor your using? (or at least the non-plugin equiped vscode)?

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "discord.js": "^12.2.0",
     "dotenv": "^8.2.0",
+    "editorconfig": "^0.15.3",
```
Comment (bradtaniguchi) [inline package.json]: Move this to `devDependencies` since it isn't directly used by the bot

**Comment by naomi-lgbt** [comment]
Hmm, maybe not. I tested it out more thoroughly and it does not seem to automatically change my linebreak to LF.

File: package.json (hunk 1 above)
Comment (JoshQuaintance) [reply package.json]: Ok

**Comment by JoshQuaintance** [comment]
> Hmm, maybe not. I tested it out more thoroughly and it does not seem to automatically change my linebreak to LF.

You need to Save. It changes stuff on save.

**Review by bradtaniguchi — APPROVED** [review]
If this works on save, and works for those on other systems (I'm on Linux), then should be good!

**Pull request merged**